### ISSUE TITLE
Bind mounts fail when Docker is running on remote host

### ISSUE THREAD

**Issue by gabegorelick** [garris/BackstopJS#858]
When Docker is running on a remote host, e.g. in an environment like [CircleCI](https://circleci.com/docs/2.0/building-docker-images/), the bind mount used to run the `backstopjs` docker container will fail:

> Error response from daemon: invalid mount config for type "bind": bind source path does not exist.

https://github.com/garris/BackstopJS/blob/61c18a6bed03b506c76aa0f23e116b7c3ae70a32/core/util/runDocker.js#L20

The recommended solution in such a case is typically to use volumes instead. Is there a reason backstopjs does not use volumes?

**Comment by garris** [comment]
@gabegorelick yup.  I don’t remember why we switched it — but I suspect that reason is probably not necessary anymore.   I would pull your change if you want to refactor that line so it would work in CI env.   I would just ask that you document the change in the PR and just explain a little bit about circle CI requirements and why your change fixes it.

Also — as an extra credit — parameters are getting duplicated.  Would be great to fix that by say, allowing CLI params to override passed config params.  Again this is just extra credit — no need to implement if you just want to fix compatibility.

Cheers!
